diff --git a/LUFA/Drivers/USB/Class/Common/AudioClassCommon.h b/LUFA/Drivers/USB/Class/Common/AudioClassCommon.h
index e9dd5d2f..a3371061 100644
--- a/LUFA/Drivers/USB/Class/Common/AudioClassCommon.h
+++ b/LUFA/Drivers/USB/Class/Common/AudioClassCommon.h
@@ -58,11 +58,6 @@
 			extern "C" {
 		#endif
 
-	/* Preprocessor Checks: */
-		#if !defined(__INCLUDE_FROM_AUDIO_DRIVER)
-			#error Do not include this file directly. Include LUFA/Drivers/USB.h instead.
-		#endif
-
 	/* Macros: */
 		/** \name Audio Channel Masks */
 		/**@{*/
diff --git a/LUFA/Drivers/USB/Core/StdDescriptors.h b/LUFA/Drivers/USB/Core/StdDescriptors.h
index c574c782..02f55885 100644
--- a/LUFA/Drivers/USB/Core/StdDescriptors.h
+++ b/LUFA/Drivers/USB/Core/StdDescriptors.h
@@ -50,20 +50,14 @@
 #define __USBDESCRIPTORS_H__
 
 	/* Includes: */
-		#include "../../../Common/Common.h"
-		#include "USBMode.h"
-		#include "Events.h"
+		#define ATTR_PACKED __packed
+		#define CPU_TO_LE16(x) (x)
 
 	/* Enable C linkage for C++ Compilers: */
 		#if defined(__cplusplus)
 			extern "C" {
 		#endif
 
-	/* Preprocessor Checks: */
-		#if !defined(__INCLUDE_FROM_USB_DRIVER)
-			#error Do not include this file directly. Include LUFA/Drivers/USB/USB.h instead.
-		#endif
-
 	/* Public Interface - May be used in end-application: */
 		/* Macros: */
 			/** Indicates that a given descriptor does not exist in the device. This can be used inside descriptors
